src/Language/Haskell/Liquid/Liquid.hs

```diff
@@ -84,7 +85,7 @@ checkTargets :: Config -> MbEnv -> [FilePath] -> IO (ExitCode, MbEnv)
 checkTargets cfg  = go 
   where
     go env []     = return (ExitSuccess, env)
-    go env (f:fs) = do whenLoud $ colorPhaseLn Loud ("[Checking: " ++ f ++ "]") ""
+    go env (f:fs) = do -- whenLoud $ colorPhaseLn Loud ("[Checking: " ++ f ++ "]") ""
```

Comment:
These changes exist exclusively because I couldn't figure out how to make GHC be less noisey, and it was easier than working around the noise.  If the reviewer knows more about this, I would be glad to know.